starky/src/f3g.rs

```diff
@@ -619,15 +643,20 @@ impl Display for F3G {
 }
 
 impl F3G {
-    fn as_bytes(&self) -> &[u8] {
-        let self_ptr: *const Self = self;
-        unsafe { slice::from_raw_parts(self_ptr as *const u8, Self::ELEMENT_BYTES * self.dim) }
-    }
+    pub const ZERO: Self = Self {
+        cube: [Fr::ZERO, Fr::ZERO, Fr::ZERO],
+        dim: 1,
+    };
+    pub const ONE: Self = Self {
+        cube: [Fr::ONE, Fr::ZERO, Fr::ZERO],
+        dim: 1,
+    };
 }
```

Comment:
Redunction. Has been defined in `FieldExtension`.

Reply:
the definition of one() is not equal to the definition of ONE which has a difference in the property of dim( one().dim=3 and ONE.dim=1 )

Reply:
The const `ZERO` has been twice in `F3G`, one is in `impl FieldExtension for F3G`, the other is here.
So remove this one will be better.

So does `F5G`.